meta_review/models.py

```diff
@@ -95,3 +95,12 @@ class Reaction(models.Model):
     giver = models.ForeignKey(Participant, related_name='give', null=True)
     receiver = models.ForeignKey(Participant, related_name='receive', null=True)
     review = models.ForeignKey(Comment, null=True)
+
+    def meta_review_add_activity(self):
```

Comment:
meta-review activity shouldn't be defined in the meta-review app, but in the gamification app.

Reply:
Will change it. Apart from that is everything fine?

Reply:
I think this is the major problem. Once the logic gets correct then we can focus on minor issues like coding style, etc.